ferrocene/tools/pull-subtrees/pull.py

```diff
@@ -385,6 +385,7 @@ def run_capture(*args, **kwargs):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--target", help="branch this pull is targeting", default=None)
+    parser.add_argument("--subtree", help="subtree to pull", default=None)
```

Comment:
when will we use this flag?

Reply:
@Veykril how will this be used?

Reply:
When you don't want to pull all subtrees at once